src/screens/DeviceInfoScreen.tsx

```diff
@@ -41,7 +39,7 @@ export type DeviceInfoScreenProps = {
   model: string
   manufacturer: string
   version: string
-  updateModule: (BasicInfoContainerState) => void
+  updateModule: (Object) => void
```

Comment:
A cicrular dependency, was proofing difficult to resolve and didn't want to spend time on this. Seems circular dependencies were only warnings and not related to the ticket bug so can revert this if needed.

Reply:
Circular dependencies caused by importing types can be fixed by using `import type` instead of `import`.